tendrl/commons/message.py

```diff
@@ -153,7 +153,7 @@ def serialize_message(obj):
         serial = obj.isoformat()
         return serial
     elif isinstance(obj, Exception):
-        return yaml.safe_dump(obj)
+        return yaml.safe_dump(obj.message)
```

Comment:
Actually what here is happening is we are passing exception object and for serialization we are taking exception message only. I feel we have to keep entire exception object, that why i gave yaml.safe_dump(obj). what is your opinion @anmolbabu